src/forms/EditableProjects.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import gql from 'graphql-tag';
+import { graphql, compose } from 'react-apollo';
+import { Link } from 'react-router-dom';
+import Chip from '@material-ui/core/Chip';
+import EditableList from './EditableList';
+import './EditableProject.css';
+
+const allProjectsQuery = gql`
+  query allProjects {
+    allProjects {
+      id
+      name
+    }
+  }
+`;
+
+const createProjectQuery = gql`
```

Comment:
Let's not allow the creation of projects through this dropdown. We specifically want to limit the amount of projects that are created